Integrations/integration-IntSights.yml

```diff
@@ -832,10 +844,76 @@ script:
         })
 
 
+    def get_mssp_sub_accounts():
+        account_id = demisto.getParam('credentials')['identifier']
+        accounts = req('GET', 'public/v1/mssp/customers').json()
+        demisto.results({
+            'Type': entryTypes['note'],
+            'EntryContext': {'IntSights.MsspAccounts': accounts},
+            'HumanReadable': tableToMarkdown('IntSights MSSP accounts for ' + account_id, [a for a in accounts], ["_id", 'CompanyName', "Status"]),
```

Comment:
Maybe map the accounts array for human readable headers (`_id` -> `ID`)
also make sure you use single quote with strings.

Reply:
I didn't want to do that because it is information straight from IntSights. So, Displaying their conventions makes it more clear as to where the information is from (how can you know the ID is not a Demisto ID?)

Reply:
Is it displayed like this in the IntSights UI? Anyway, for context there is also a description for each output. It just looks better this way IMO.